modules/address/components/AddressDelegatedTo.tsx

```diff
@@ -16,6 +14,7 @@ import AddressIconBox from './AddressIconBox';
 import { parseUnits } from 'ethers/lib/utils';
 import { formatValue } from 'lib/string';
 import { DateWithHover } from 'modules/app/components/DateWithHover';
+import EtherScanLink from 'modules/web3/components/EtherScanLink';
```

Comment:
Not a big deal but Etherscan doesn't use camelcasing
<img width="190" alt="Screen Shot 2022-09-28 at 9 15 04 AM" src="https://user-images.githubusercontent.com/5225766/192713193-71799590-7f49-4b48-831c-0de249a5d958.png">